test/rosdep_repo_check/README.md

```diff
@@ -15,3 +15,13 @@ To review the entire rosdep database and not only the rules that were changed in
 
 PYTHONPATH=test python3 -m rosdep_repo_check
 ```
+
+## Adding new repository checks
+
+Platform checks can be added by updating [config.yaml](./config.yaml).
+
+* `package_sources` contains a set of repository base urls for each operating system distribution
+* `package_dashboards` contains an optional list of matching repository url patterns and template urls which can be used to extract and compose web links to packages in the matching distributions.
+* `supported_versions` lists of operating system versions or codenames that checks should be checked with the package sources
```

Comment:
Would you mind re-phrasing this?

Reply:
Also, it might be worth noting that the last one on the list is used for trying to generate a 'suggestion'.

Reply:
Thanks for highlighting. That was definitely a case where I wrote the first and second halves of the sentence separately.